core/README.md

```diff
@@ -30,11 +84,6 @@ navigation.
             "type": "application/json",
             "href": "https://stacserver.org"
         },
-        {
```

Comment:
conformance, data, and search are not part of Core, so removed them from this example.